.gitignore

```diff
@@ -8,4 +8,4 @@ __pycache__/
 .env
 logs/
 app/app.db
-requirements.txt
+dump.rdb
```

Comment:
dump.rdb is Redis cache file so it can be ommited being commited to repository
In more detail:
https://redis.io/docs/management/persistence/